### DISCUSSION TITLE
Rewrite folder name on different languages

### DISCUSSION THREAD

**Discussion by Yanick64** [vercel/next.js#20896] [Help]
I have a multi-language website with NextJS v10 and I am trying to translate a folder to be different on each language.

My folder structure looks like the this:

pages/kennis/index.tsx
I can't figure out how to translate the "kennis" folder to be different on each language. The documentation of NextJS (https://nextjs.org/docs/advanced-features/i18n-routing#domain-routing) does not say anything about this.

Let's say for NL I want "/kennis" and for EN I want "/en/knowledge".

Does someone know a solution for this?

**Comment by dmudro** [comment]
There a recently opened issue about this; see https://github.com/vercel/next.js/issues/20665

**Reply by svenve-git** [reply]
isn't there a difference, since we're talking about folder name vs page name?

**Reply by dmudro** [reply]
`pages/kennis/index.tsx` and `pages/kennis.tsx` are equivalents. I imagine the solution should cover both
